blocks/library/more/index.js

```diff
@@ -39,6 +46,28 @@ export const settings = {
 		},
 	},
 
+	transforms: {
+		from: [
+			{
+				type: 'raw',
+				isMatch: ( node ) => node.dataset.block === 'core/more',
```

Comment:
So the `dataset` is used here and it's not a generic API right? it's just a specific attribute created by the `special-comment-converter.js` which is also specific to this bloc.

This makes me wonder, if this converter's code shouldn't be written somewhere in this transform instead?

Reply:
> it's not a generic API right? it's just a specific attribute created by the special-comment-converter.js which is also specific to this bloc.

Correct. It is a bit of a fake separation. However, I named it `special-comment-converter` instead of `more-converter` because I'm anticipating its use for `<!--nextpage-->` later.

> This makes me wonder if this converter's code shouldn't be written somewhere in this transform instead?

Hm. Maybe I looked at it the wrong way, but basically dataset + custom tag was what I came up with to _preserve_ the more tag and not let it get destroyed by the rest of `rawHandler`. This seems like a better trade-off than e.g. making exceptions in the other transformers to ignore `more`. What do you think? Also pinging @iseulde for ideas. :)

Reply:
I agree that the separation is weird, but can't think of anything better right now... Hm or maybe? The raw handler itself is supposed to return clean HTML which blocks can hook into to transform. We already have shortcodes in this mix too though, but the transform is `shortcode` instead of `raw`. Maybe we could follow the same pattern and have a `comment` transform? If no block transforms the comment, then we should just drop the comment (not pass through `raw` transforms). What do you think? This removes the need for the special case comment converter.